### ISSUE TITLE
How to render the image based on the predict of confidence > 0.5?

### ISSUE THREAD

**Issue by DanielChuDC** [ultralytics/yolov5#5257]
## ❔Question

Hi , thank you for this amazing project. I would like to ask how are we able to label the box only if the confidence of the prediction more than certain value?

Currently using the code `results.render()` , it would be nice if can pass in `results.render(0.5)` to set the box.

Thank you for your time!

## Additional context
For example, in this image, have multiple boxes. Would like to render the box only the confidence > 0.5
<img width="1018" alt="Screenshot 2021-10-19 at 8 37 59 PM" src="https://user-images.githubusercontent.com/52316624/137910672-c6252b17-e822-46ef-b35c-f03331070764.png">

**Comment by glenn-jocher** [comment]
@DanielChuDC YOLOv5 **PyTorch Hub** tutorial explains how to set confidence threshold:
### Inference Settings
Inference settings such as **confidence threshold**, NMS **IoU threshold**, and **classes** filter are model attributes, and can be modified by:
```python
model.conf = 0.25  # confidence threshold (0-1)
model.iou = 0.45  # NMS IoU threshold (0-1)
model.classes = None  # (optional list) filter by class, i.e. = [0, 15, 16] for persons, cats and dogs

results = model(imgs, size=320)  # custom inference size
```

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️ RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Supervisely Ecosystem](https://docs.ultralytics.com/yolov5/tutorials/model_export8)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TorchScript, ONNX, CoreML Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)

**Comment by DanielChuDC** [comment]
Thank you! @glenn-jocher 

My bad did not read carefully!

Closing issue now
